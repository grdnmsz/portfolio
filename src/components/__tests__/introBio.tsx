import React from 'react'
import renderer from 'react-test-renderer'

import { IntroBio } from '../'

describe('IntroBio', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<IntroBio />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
