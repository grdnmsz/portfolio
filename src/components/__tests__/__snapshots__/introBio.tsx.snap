// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroBio should renders correctly 1`] = `
<div
  className="relative z-10 pb-8 bg-white sm:pb-16 md:pb-20 lg:max-w-2xl lg:w-full lg:pb-28 xl:pb-26"
>
  <h1
    className="text-4xl tracking-tight font-extrabold sm:text-5xl md:text-6xl"
  >
    <span
      className="block animate-fade xl:inline"
    >
      Hi,
      <br />
      I am Gordon!
    </span>
  </h1>
  <p
    className="mt-3 animate-fade text-base sm:mt-5 sm:text-lg sm:max-w-xl  md:mt-5 lg:mx-0"
  >
    I am a Software Engineer looking for new opportunities in Full Stack development. Currently unemployed, Iâ€™m available and motivated to take on creative and challenging missions.
  </p>
  <div
    className="mt-5 sm:mt-8 sm:flex lg:justify-start"
  >
    <div
      className="rounded shadow"
    >
      <a
        className="w-full flex items-center justify-center px-8 py-3 text-base font-medium rounded bg-headerColor hover:bg-teal-700 md:py-4 md:text-lg md:px-10"
        href="mailto:gordonmes@gmail.com"
      >
        Get in touch
      </a>
    </div>
    <div
      className="mt-3 sm:mt-0 sm:ml-3"
    >
      <div
        className="rounded shadow"
      >
        <a
          className="w-full flex items-center justify-center px-8 py-3 text-base font-medium rounded bg-headerColor hover:bg-teal-700 md:py-4 md:text-lg md:px-10"
          href="/404"
        >
          Download Resume
        </a>
      </div>
    </div>
  </div>
</div>
`;
